src/NUnitFramework/nunitlite.runner/TextUI.cs

```diff
@@ -342,20 +342,25 @@ public void DisplaySummaryReport(ResultSummary summary)
                 SkipLine();
 
             WriteSectionHeader("Test Run Summary");
-            WriteLabelLine("   Overall result: ", overallResult, overallStyle);
+            WriteLabelLine("  Overall result: ", overallResult, overallStyle);
 
-            WriteSummaryCount("   Tests run: ", summary.RunCount);
+            WriteSummaryCount("  Tests run: ", summary.RunCount);
             WriteSummaryCount(", Passed: ", summary.PassCount);
-            WriteSummaryCount(", Errors: ", summary.ErrorCount, ColorStyle.Error);
-            WriteSummaryCount(", Failures: ", summary.FailureCount, ColorStyle.Failure);
+            WriteSummaryCount(", Failed: ", summary.FailedCount);
             WriteSummaryCount(", Inconclusive: ", summary.InconclusiveCount);
+            WriteSummaryCount(", Skipped: ", summary.SkipCount);
+            _outWriter.WriteLine();
+
+            _outWriter.Write("    Failed Tests - "); 
+            WriteSummaryCount("Failures: ", summary.FailureCount);
+            WriteSummaryCount(", Errors: ", summary.ErrorCount);
+            WriteSummaryCount(", Invalid: ", summary.InvalidCount);
             _outWriter.WriteLine();
```

Comment:
This group of lines could be conditional on having some Failed tests.
If (summary.FailedCount > 0)...